pico/tests/interception/src/main/java/io/helidon/pico/tests/interception/ReturningInterceptor.java

```diff
@@ -37,6 +37,7 @@ static Invocation lastCall() {
         return LAST_CALL.getAndSet(null);
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Curiously the SuppressWarnings("ALL") doesn't catch the unchecked vararg warnings in the generated interceptor - this did the trick though.